containers/container-tests/src/test/java/org/hawkular/containers/TestCassandraScaling.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2014-2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.containers;
+
+import static org.junit.Assert.assertEquals;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.fabric8.kubernetes.api.model.ReplicationController;
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author mwringe
+ */
+@RunWith(Arquillian.class)
+public class TestCassandraScaling extends BaseContainerTests {
+
+
+    @Test
+    public void testScaling() throws Exception {
+        // Test with the intial setup of 1 replica
+        testWriteData("initial", "11.102", 1);
+        ReplicationController rc = client.getReplicationController(CASSANDRA_NAME);
+
+        // Test scaling up to 2 replicas
+        rc.getSpec().setReplicas(2);
+        client.updateReplicationController(CASSANDRA_NAME, rc);
+
+        while(client.getPodsForService(CASSANDRA_NAME).size() < 2) {
```

Comment:
When the size it two, it means that there there are 2 Cassandra pods running. It does not necessarily mean that the Cassandra node is fully started or not.
I have looked into making it so that a node stays in the 'pending' state until it finishes bootstrapping, but then it means we can no longer discover the other nodes in the cluster. Its something we need to fix.